tests/haskell-utils-tests.el

```diff
@@ -1,102 +1,179 @@
+;;; haskell-utils-tests.el --- Tests for Haskell utilities package
+
+;; Copyright © 2016 Athur Fayzrakhmanov. All rights reserved.
```

Comment:
Please don't treat this as self-praise or etc. this is GPL requirement.  Also everyone who contribute to haskell-mode files should list themselves as authors.